apps/condo/pages/billing/index.tsx

```diff
@@ -43,12 +44,14 @@ const BillingPage = () => {
             <PageWrapper>
                 <PageHeader title={<Typography.Title style={{ margin: 0 }}>{PageTitle}</Typography.Title>}/>
                 <TablePageContent>
-                    <BillingPageContent
-                        access={canReadBillingReceipts}
-                        contextLoading={contextLoading}
-                        contextError={contextError}
-                        context={currentContext}
-                    />
+                    <AmplitudeAuthorizedUser>
```

Comment:
It's unclear, where to use `AmplitudeAuthorizedUser` and where to use `TrackPageLoadEvent` component.
First, second or both…
Is it possible to incapsulate all the logic in one component, — new or inside of `PageWrapper`?